src/sdk.js

```diff
@@ -114,9 +118,12 @@ export default class AccountsSDK {
   authorizeURL(options = {}, flow = '') {
```

Comment:
Can we add a new option called `preserveOriginalHref` set by default to false and run the persister logic only when explicitly set to true by the customer?